debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TestHelper.java

```diff
@@ -234,8 +234,8 @@ protected static boolean shouldSSLConnectionFail() {
         return Boolean.parseBoolean(System.getProperty(TEST_PROPERTY_PREFIX + "ssl.failonconnect", "true"));
     }
 
-    protected static int waitTimeForRecords() {
-        return Integer.parseInt(System.getProperty(TEST_PROPERTY_PREFIX + "records.waittime", "2"));
+    public static int waitTimeForRecords() {
+        return Integer.parseInt(System.getProperty(TEST_PROPERTY_PREFIX + "records.waittime", "20"));
```

Comment:
@Naros Thanks for the timeout changes! The default should still be kept to the original value.

Reply:
Also could you please undo the whitespace-only changes, @Naros!

Reply:
@gunnarmorling I can; I'm most likely going to need to reset IJ back to default and redo these format settings as its starting to be more and more problematic it seems :/

Reply:
@gunnarmorling Are you referring to the changes in `PostgresConnectorIT`?

Reply:
@Naros I really meant all the changes in this PR which are only white-space changes (indentation). If I set this option in the PR view here on GH, most of the changes are hidden, i.e. the PR contains many superfluous changes. If it's too much of a hazzle to undo, let's talk.